Wordle.Api/Wordle.Api/Services/WordleService.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
```

Comment:
it be a good idea to try and implement tests for this.

Reply:
Tests kept giving me weird errors that didn't make sense.